src/utils/api.js

```diff
@@ -22,3 +22,16 @@ export function getComments(article_id) {
       return data.comments;
     });
 }
+export function patchVotes(article_id) {
```

Comment:
This function only increases the vote, but you also have a downvote button. Have a think about how to make this function more generic by maybe passing in an argument that controls if it's an upvote or downvote